### PR TITLE
Use 'originIdentifier' in getUserMedia() steps (issue #339)

### PR DESCRIPTION
**Pull request by adam-be**
Fix for issue #339

First commit is purely editorial so please review the second.

### CODE REVIEW

File: getusermedia.html [hunk 1]
```diff
@@ -2797,11 +2797,11 @@
                   browsing context</a>'s origin.</p>
                 </li>
                 <li>
-		  <p><a>Request permission</a> for use of the devices,
-                  while considering
-		  all devices attached to a <code>live</code> MediaStreamTrack
-		  to have permission status "granted", resulting in a set of
-		  provided media.</p>
+                  <p>For the origin identified by <var>originIdentifier</var>,
+                  <a>request permission</a> for use of the devices, while
```
Comment (jan-ivar) [inline getusermedia.html]: I don't see how we're expected to verify anything about a "request permission" algorithm that doesn't exist yet.

Comment (dontcallmedom) [inline getusermedia.html]: https://w3c.github.io/permissions/#request-a-permission ?

File: getusermedia.html [hunk 2]
```diff
@@ -2797,11 +2797,11 @@
...
+                  considering all devices attached to a <code>live</code>
+                  MediaStreamTrack to have permission status "granted",
+                  resulting in a set of provided media.</p>
```
Comment (jan-ivar) [inline getusermedia.html]: I also don't know when this language snuck in, but it doesn't match how Firefox works. The existence of live tracks has no bearing on whether users get prompted by `getUserMedia` in Firefox. You get prompted for each track you request, and that's it.

What was the point of introducing this language? Was there any discussion?

File: getusermedia.html (hunk 1 above)
Comment (jan-ivar) [inline getusermedia.html]: Thanks, it's not in the TR which is what we link to, and we still have a [disclaimer](https://rawgit.com/adam-be/mediacapture-main/use-origin-identifier/getusermedia.html#request-permission) in our doc saying it doesn't exist. The link then seems like it should be ***request a permission***, not ***request permission***.

Also, maybe everyone else is aware, but this surprised me, that the other spec seems to follow a slightly different process for consensus than at least I was used to. Specifically, the editor of the other spec [says](https://github.com/w3c/permissions/issues/85#issuecomment-213616988) he doesn't "claim that the current text represents any sort of consensus."

I'm surprised we're adding such dependencies when we're supposed to be way past last call.

Comment (jan-ivar) [inline getusermedia.html]: Is there a way in this spec now for `getUserMedia` to succeed without granting persistent permission? What does ***request a permission*** return in that case?

See https://github.com/w3c/mediacapture-main/issues/350 for my take down of the *request permission* algorithm.

File: getusermedia.html (hunk 2 above)
Comment (jan-ivar) [inline getusermedia.html]: Also, doesn't this literally say site B must be granted access if site A currently has a live MediaStreamTrack?

**Comment by adam-be** [comment]
@jan-ivar, my interpretation is that your concerns are mainly with text that's not added by this PR (that you've also created a new issue for). And that shouldn't block us from merging this PR. Is that correct?

**Comment by jan-ivar** [comment]
@adam-be Sounds like the algorithm referenced should be [request ***a*** permission](https://w3c.github.io/permissions/#request-a-permission), though I see no evidence that it takes an `origin` (though the document does talk about origin elsewhere now).

The level of meaninglessness seems neither increased nor diminished by this PR, so I have no problem with it merging.

**Pull request merged**